syntax="proto3";
package greeting;

message GreetRequest{
    string message = 1;
}

message GreetResponse{
    string message = 1;
}

service GreetService{
    rpc SayHello (GreetRequest) returns (GreetResponse){}
}